client/superdesk.config.js

```diff
@@ -6,7 +6,8 @@
  */
 module.exports = function(grunt) {
     return {
-        apps: [],
+        apps: ['fidelity.superdesk'],
+        importApps: ['../fidelity'],
```

Comment:
imo it should be `'./fidelity'`

Reply:
Just tried it. Looks like `dist` is the base path, so it won't work with `'./fidelity'`

Reply:
I see, ok